api/client/start.go

```diff
@@ -49,6 +49,7 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 	cmd := Cli.Subcmd("start", []string{"CONTAINER [CONTAINER...]"}, Cli.DockerCommands["start"].Description, true)
 	attach := cmd.Bool([]string{"a", "-attach"}, false, "Attach STDOUT/STDERR and forward signals")
 	openStdin := cmd.Bool([]string{"i", "-interactive"}, false, "Attach container's STDIN")
+	detachKeys := cmd.String([]string{"-detach-keys"}, "", "Specify the escape key sequence used to detach a container")
```

Comment:
Hum it appears it's too long for one line and it won't allow me multi-line option "comment" (see DockerSuite.TestHelpTextVerify). So… I don't know what to put there :stuck_out_tongue_closed_eyes: